WordPress/Classes/ViewRelated/Post/PostListEditorPresenter.swift

```diff
@@ -40,28 +40,56 @@ struct PostListEditorPresenter {
+    /// A dialog giving the user the choice between loading the local version or the remote version when there's a version conflict
+    private static func autosaveOptionsViewController(for post: Post, didTapOption: @escaping (_ loadAutosaveRevision: Bool) -> Void) -> UIAlertController {
+        let title = NSLocalizedString("Resolve sync conflict", comment: "Title displayed in popup when user has to resolve a verion conflict due to unsaved/autosaved changes")
+
+        let localFormattedDate = dateAndTime(for: post.dateModified)
+        let remoteFormattedDate = dateAndTime(for: post.autosaveModifiedDate)
+
+        let alertMessageText = """
+    This post has two versions that are in conflict. Select the version you would like to keep:
+
+    Local
+    Saved on \(localFormattedDate)
+
+    Remote
+    Saved on \(remoteFormattedDate)
 
-        let title = NSLocalizedString("Which version would you like to edit?", comment: "Title displayed in popup when user has the option to load unsaved changes")
+    """
 
-        let saveDateFormatted = dateAndTime(for: saveDate)
-        let autosaveDateFormatted = dateAndTime(for: autosaveDate)
-        let message = String(format: NSLocalizedString("You recently made changes to this post but didn't save them. Choose a version to load:\n\nFrom this device\nSaved on %@\n\nFrom another device\nSaved on %@\n", comment: "Message displayed in popup when user has the option to load unsaved changes. \n is a placeholder for a new line, and the two %@ are placeholders for the date of last save on this device, and date of last autosave on another device, respectively."), saveDateFormatted, autosaveDateFormatted)
+        let message = NSLocalizedString(alertMessageText, comment: "Message displayed in popup displayed to resolve version conflict")
 
-        let loadSaveButtonTitle = NSLocalizedString("From this device", comment: "Button title displayed in popup indicating date of change on device")
-        let fromAutosaveButtonTitle = NSLocalizedString("From another device", comment: "Button title displayed in popup indicating date of change on another device")
+        let localButtonTitle = NSLocalizedString("Keep Local", comment: "Button title displayed in popup indicating date of change on device")
+        let remoteButtonTitle = NSLocalizedString("Keep Remote", comment: "Button title displayed in popup indicating date of change on another device")
 
         let alertController = UIAlertController(title: title, message: message, preferredStyle: .alert)
-        alertController.addAction(UIAlertAction(title: loadSaveButtonTitle, style: .default) { _ in
-            didTapOption(false)
+        alertController.addAction(UIAlertAction(title: localButtonTitle, style: .default) { _ in
+            if post.hasVersionConflict {
```

Comment:
This logic will be moved out in to a separate class `VersionConflictResolver`. It was originally put in to `PostListEditor` on the incorrect assumption we react to both conflicts in the same way. Now we react differently it would be a bad design to keep these in the same place.